README.md

```diff
@@ -1,17 +1,38 @@
-# Sample
+﻿# ABOUT
 
-This module is for fun and trying out new crazy and wacky things with the game. It houses a number of example blocks and UI screens.
+Sample.
 
-### Credits for images:
+This module is for fun and trying out new crazy and wacky things with the game. 
+It houses a number of example blocks and UI screens.
 
-WhiteStone: http://www.planetminecraft.com/texture_pack/new-start---by-loufisch/
+ 
+A person can test their UI screens and other various creations in this module.
+This module is open to local developments and tests.
+It is more inclined towards developers who want to test stuff; but players can enjoy it too!
```

Comment:
Convert these 3 lines (9 to 11) into a single line.

Reply:
Replace the word `inclined` with `focused` - we usually use `inclined` with living things, like `i'm inclined to sleep`. Also, replace the `;` with `,`